functions/vercel/api/query/[table].ts

```diff
@@ -0,0 +1,66 @@
+import type { VercelRequest, VercelResponse } from '@vercel/node'
+import type { TableNames } from '../../_lib/types'
+import 'isomorphic-fetch'
+import { getXataClient } from '../../_lib/xata.codegen'
+
+export default async function handler(
+  request: VercelRequest,
+  response: VercelResponse
+) {
+  const xata = getXataClient()
+  const table = request.query.table as TableNames
+
+  const { filter, sort } = request.body ?? {}
+
+  const hasFilter = Boolean(filter)
+  const hasSort = !!sort
+
+  if (request.method !== 'POST') {
+    response.status(405).end('Only `POST` requests')
```

Comment:
I belive there's a missing `return` here. Otherwise the execution of the function will continue even if you have _ended_ the response.

Reply:
Good point! Will do!

I was only caring about the response!
Thanks!